bids-validator/validators/json/schemas/microscopy.json

```
@@ -62,12 +62,13 @@
     "InstitutionName",
     "InstitutionAddress",
     "InstitutionalDepartmentName",
-
     "BodyPart",
+    "BodyPartDetails",
     "Environment",
```

```suggestion
    "SampleEnvironment",
```
I just realized that `Environment` was changed to `SampleEnvironment` a while ago and we forgot to change it here.

---

Done